presto-native-execution/presto_cpp/main/connectors/PrestoToVeloxConnectorUtils.h

```diff
@@ -46,8 +47,8 @@ std::vector<velox::common::Subfield> toRequiredSubfields(
 velox::common::CompressionKind toFileCompressionKind(
     const protocol::hive::HiveCompressionCodec& hiveCompressionCodec);
 
-velox::dwio::common::FileFormat toVeloxFileFormat(
-    const presto::protocol::hive::StorageFormat& format);
+velox::connector::hive::HiveColumnHandle::ColumnType toHiveColumnType(
```

Comment:
I would've liked to keep this file independent of all the Hive specific conversions. But seems like there are multiple of them in this file (toHiveTableHandle) etc . We could move them to their own separate file as well. What do you think of adding a HivePrestoToVeloxConnectorUtils as well ?

@PingLiuPing